js/packages/opentelemetry-plugin-launchdarkly-node-server/package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "@lightstep/opentelemetry-plugin-launchdarkly-node-server",
-			"version": "0.0.10",
```

Comment:
Also, undo these changes on js files, It would be the best if we don't touch them since we didn't change anything there.